src/rules/selector-attribute-square-brackets-space-inside/README.md

```diff
@@ -0,0 +1,93 @@
+# selector-attribute-square-brackets-space-inside
+
+Require a single space or disallow whitespace on the inside of the square brackets within selectors.
```

Comment:
"Require a single space or disallow whitespace on the inside of the brackets within attribute selectors."